tensorflow/c/c_api.cc

```diff
@@ -1667,6 +1667,25 @@ void TF_DeleteGraph(TF_Graph* g) {
   if (del) delete g;
 }
 
+size_t TF_GraphOperationsCount(TF_Graph* graph) {
```

Comment:
Ah, right. Good catch!
I suppose we could have the native method operate on the `OperationIterator` and edit the fields in-place?  Or perhaps return an object representing a pair of integers - the operation handle and the new value of pos?